.github/pull_request_template.md

```diff
@@ -1,9 +1,5 @@
 # Overview
 
-## Links to jira tickets
-
-- {Include links to all of the applicable Jira tickets}
```

Comment:
This does let us account for multiple tickets, which the jira action doesn't.

Reply:
Yeah was kinda thinking that but thought those prs it isn't super important to link the ticket to - but wanted to get thoughts :)

Happy to keep this, was more to start a discussion. Will close this PR for now 😇